src/app/component/MissionCard.jsx

```diff
@@ -53,7 +60,7 @@ const MissionCardContent = ({ classes, contentItems }) => (
  * @component
  */
 const MissionCard = withStyles(styles)(({ children, classes, mission, ...rest }) => {
-  const title = mission.title;
+  const title = mission.title ? mission.title : "No title supplied.";
```

Comment:
Pattern related: This should be syntatically similar to the pattern already established below https://github.com/factn/resilience-app/pull/411/files#diff-5261dfc248a50fac298f589e95348d05R65
```
mission?.title || "No title supplied."
```

Bigger question: Should this be a required field? Can a mission be created without a title?

React suggestion: We can leverage [defaultProps](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) for this component. We already have the proptypes established below, so something like
```
MissionCard.defaultProps = {
  mission: {
    title: "No title supplied",
    status: "",
    pickUpLocation: {
      address: "No Data"
    }
  }
};
```

Default props will be the assigned values to props unless one is given during component creation

Reply:
This can clean up some of the logic in this component

Reply:
Hi @jwu910 , I have added your suggestions.  Although it seems fairly obvious to me that "title" should be a required field, there seems to be tons of data in firebase without a title, making me wonder if it's possible to set a not-empty constraint at that level.  @mat10tng manages the models and could better speak to this requirement.

Reply:
Yeah would be good to discuss. We can always program defensively at least on the build time with typescript and/or utilizing propTypes for runtime errors to catch this stuff.

Reply:
My 2c on 'title' for missions is that they never made sense in the first place. 

A designer added it to a sketch at some point and it got copied everywhere but imho nobody stopped to ask who the heck is going to want to stop long enough to give a mission a 'title'? 

I worked with Karen to revise the designs and we tried to 'squash' the idea of a title on missions at that point but I don't know if maybe the idea lives on in the code. Like a zombie wandering the moors. 

Far as I am concerned the <MissionType> plus <Summary of mission details> can be used wherever a mission 'title' might be needed. 

For instance if its a food box mission the UI shows "FoodBox' and "3 Vegetable Fruit Medly" or just Foodbox: 3 depending on exact context. 

If its a 'generic errand' we may want to show just the first 40 characters of the full 'notes' field as a summary (that can be clicked on for more detail). But since generic errand is not in MVP0 anymore, we can cross that bridge when we come to it.

This doesn't relate to mission props but just title. 

I know that the code has diverged from the designs but I suggest we refer back to the designs and designers in terms of this 'title' thing. 

Also, related, the idea of having an 'image' for every mission. Nice in theory but doesn't make sense till after they are delivered. So.. think that's gone in the latest designs also.

Reply:
FWIW MissionType schema 
https://github.com/factn/resilience-app/blob/5930fb6164747cd58e4d40d078300149e92be8be/src/app/model/schema.ts#L118

But in fact we only supporting one mission type for MVP.0 - MissionType.foodbox

So i guess a magic string 'foodbox' is OK. But better to use the enum..